app/js-src/backend.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+import { Client } from "rpc-websockets"
+let ws = new Client('ws://localhost:8080')
+import * as wn from 'webnative'
+
+let backend = {
```

Comment:
nit: 'backend' doesn't tell me enough about what this *is*, either here or in the file name. What is the thing this talks to? Where is it documented? :)